average.js

```diff
@@ -1,6 +1,7 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  const num2 = numbers.filter( value => !Number.isNaN(value) );
```

Comment:
The idea to name variables is to let readers know their intent without having to read the entire code.
Try a descriptive name for `num2` to express its intent